src/object-properties.js

```diff
@@ -108,8 +108,13 @@ const properties = {
  * @property {string=} [value=''] - To select certain values in a field or set a variable. Required for types 'selectValues', 'selectMatchingValues', 'toggleSelect' and 'setVariable'
  * @property {string=} [variable=''] - Name of variable. Required for type 'setVariable'
  * @property {boolean=} [partial=false] - Set to true if you want to do a partial reload.
- * @property {string=} [automation=''] - ID of the automation. Required for type 'setVariable'
+ * @property {string=} [automation=''] - Item ID of the automation. Required for type 'executeAutomation'
+ * @property {string=} [automationId=''] - ID of the automation. Required for type 'executeAutomation'
```

Comment:
I don't understand this. Are these the same thing? Hard to determine from the description. If not, are both of them are required for executeAutomation?

In general, I would strongly advice against renaming existing properties. I know from experience that handling different versions of the properties can be really messy when customers start migrating between sense versions etc.

Reply:
I will write a better description in this file. 

The automation property which existed before was the item id coming from `api/v1/items?resourceType=automations` - saving this id in the properties is not the best thing to do as we need the automation id when triggering the automation and we were calling the API after every button click to get this information. I chose to leave the old automation prop as is and to create a new automationId prop which is using the actual automation id.

Reply:
I added better descriptions in the latest commit